src/azure-cli/setup.py

```diff
@@ -50,6 +50,7 @@
 ]
 
 DEPENDENCIES = [
+    'antlr4-python3-runtime~=4.7.2',
```

Comment:
I'm not sure. Is it the correct way to fix this issue? @fengzhou-msft  Since python2 and python3 share the same namespace.

Reply:
We do not support python 2 now, this is fine.

Reply:
Do you have any code that may have different handlings for py2 and py3 with this antlr4 dependency? You may also clean that code if yes.

Reply:
no. They share same signature to us.